consensus/vbft/node_sync.go

```diff
@@ -400,7 +401,7 @@ func (self *PeerSyncer) requestBlock(blkNum uint32) (*Block, error) {
 		return nil, err
 	}
 	self.server.msgSendC <- &SendMsgEvent{
-		ToPeer: self.peerIdx,
+		ToPeer: math.MaxUint32,
```

Comment:
if only broadcasting support, consider fix this in self.server.msgSendC handling??

Reply:
yes. I will delete some unused code,like send to peer function.